tt_metal/tools/profiler/kernel_profiler.hpp

```diff
@@ -429,13 +432,17 @@ namespace kernel_profiler{
     }
 
 
-    template<uint32_t timer_id, bool dispatch = false>
```

Comment:
The C++ police will catch you lol. We should do an enum here I am doing this in the coming PR https://github.com/tenstorrent/tt-metal/blob/a78da6ade65e449c3d83cc5196d3abd4393264a0/tt_metal/tools/profiler/kernel_profiler.hpp#L480